legistar/bills.py

```diff
@@ -270,9 +270,17 @@ def endpoint(self, route, *args):
         response = self.get(url.format(*args))
         return response.json()
 
-    topics = partialmethod(endpoint, '/matters/{0}/indexes')
     code_sections = partialmethod(endpoint, 'matters/{0}/codesections')
 
+    def topics(self, *args, **kwargs):
```

Comment:
I think it's becoming clearer and clearer that we should be moving to a more object oriented approach, but this seems like a very reasonable step right now.